gpustack/routes/models.py

```diff
@@ -54,18 +54,20 @@ async def get_models(
     search: str = None,
     categories: Optional[List[str]] = Query(None, description="Filter by categories."),
     cluster_id: int = None,
+    existing_fields: Optional[Dict[str, Any]] = None,
+    target_class: Union[Model, MyModel] = Model,
```

Comment:
Can we prevent exposing it in openAPI schema

Reply:
Updated